src/borg/archiver.py

```diff
@@ -2159,8 +2159,9 @@ def do_break_lock(self, args, repository):
             .. note::
 
                 Via ``--pattern`` or ``--patterns-from`` you can define BOTH inclusion and exclusion
-                of files using pattern prefixes ``+`` and ``-``. With ``--exclude`` and
-                ``--exlude-file`` ONLY excludes are defined.
+                of files using pattern prefixes ``+`` and ``-``, so the PATH argument of the command
+                can be omitted. With ``--exclude`` and ``--exlude-from`` ONLY excludes are defined,
+                and the PATH argument defines the includes.
```

Comment:
remove ", so the PATH argument of the command can be omitted".
also remove ", and the PATH argument defines the includes".

the PATH arguments rather define the recursion roots / starting points (so it is rather like the `R` given in a pattern file).

Reply:
Done.